alf/examples/her_fetchpush.gin

```diff
@@ -7,8 +7,8 @@ critic/CriticNetwork.observation_preprocessing_combiner=@NestConcat()
 critic/CriticNetwork.action_preprocessing_combiner=@NestConcat()
 
 ReplayBuffer.keep_episodic_info=True
-ReplayBuffer.postprocess_exp_fn=@hindsight_relabel_fn
-hindsight_relabel_fn.her_proportion=0.8
+HindsightExperienceTransformer.her_proportion=0.8
+TrainerConfig.data_transformer_ctor=[@ObservationNormalizer, @HindsightExperienceTransformer]
```

Comment:
please also add this gin to train_play_test

Reply:
done